### PR TITLE
Add jdk11 to ci matrix

### PR DESCRIPTION
**Pull request by eikek**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        java: [ 'adopt@1.8' ] # TODO: add test for adopt@1.11 and possibly adopt@1.16
+        java: [ 'adopt@1.8', 'adopt@1.11', 'openjdk@1.11' ]
```
Comment (sideeffffect) [inline .github/workflows/ci.yml:10]: How about adding 16 to the mix?

Comment (eikek) [reply .github/workflows/ci.yml:10]: I couldn't find it in the list of java versions when unfolding the scala-setup action. But I can try to add it

Comment (eikek) [reply .github/workflows/ci.yml]: It doesn't seem to be available currently: ` Error: Couldn't find Java openjdk@1.16.` Or do you think I missed something?

Comment (sideeffffect) [reply .github/workflows/ci.yml]: Yeah, it seems like there's only Zulu, so you could try that one: `zulu@1.16`

**Comment by sideeffffect** [comment]
:tada:

**Comment by eikek** [comment]
Nice it's fixed. But I suspect the cause is also on sbt "layered classloader". I still think the fix is good anyways.

**Pull request merged**